crates/api/src/site/mod_log.rs

```diff
@@ -23,15 +23,23 @@ pub async fn get_mod_log(
 
   check_private_instance(&local_user_view, &local_site)?;
 
-  let is_mod_or_admin = if let Some(local_user_view) = &local_user_view {
-    check_community_mod_of_any_or_admin_action(local_user_view, &mut context.pool())
-      .await
-      .is_ok()
+  // Only show the modlog names if:
+  // You're an admin or
+  // You're fetching the modlog for a single community, and you're a mod
+  // (Alternatively !admin/mod)
+  let hide_modlog_names = if let Some(community_id) = data.community_id {
+    is_mod_or_admin_opt(
+      &mut context.pool(),
+      local_user_view.as_ref(),
+      Some(community_id),
+    )
+    .await
+    .is_err()
   } else {
-    false
+    is_admin_opt(local_user_view.as_ref()).is_err()
```

Comment:
Its enough to use `!local_user_view.local_user.admin` directly here.

Reply:
Its optional there, so I'd prefer the function I added.

Reply:
```rust
local_user_view.map(|l| l.local_user.admin).unwrap_or_default()
```

Doesnt make sense to add a new function which isnt used anywhere else, and performs unnecessary extra checks when all we need is reading a single bool.